### PR TITLE
Start file on serving suggestions

### PR DESCRIPTION
**Pull request by bhazelton**
Added a file for serving suggestions so that we know what to do with guacamole.

### CODE REVIEW

**Comment by vsivsi** [comment]
I see your request Bryna, I will review shortly

File: serving.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Serve with tortilla chips.
```
Comment (vsivsi) [inline serving.txt:1]: Any particular kind of chips?  Nacho cheese flavor?  **_Cool Ranch?_**

Comment (bhazelton) [reply serving.txt]: I added some more detail on flavors, let me know if I should do more.

File: serving.txt [hunk 2]
```diff
@@ -1 +1 @@
-Serve with tortilla chips.
+Serve with tortilla chips. Either standard or flavored, Vaughn like Cool Ranch!
```
Comment (vsivsi) [inline serving.txt:1]: Nice update Bryna, much clearer.  Cool ranch was /s

**Pull request merged**